usr/share/rear/conf/default.conf

```diff
@@ -3847,3 +3851,12 @@ DRLM_ID="$HOSTNAME"
 # run the risk that the code in question will be removed in a future ReaR
 # release.
 DISABLE_DEPRECATION_ERRORS=()
+
+####
+# Include a Python interpreter
+#
+# default is false, can be set to enable finding a Python interpreter or to
+# specify the path to the Python binary to use
+PYTHON_INTERPRETER=false
+# set to true to include only the Python interpreter without and dist or site packages
```

Comment:
Could there be a typo and
"without any dist or site packages"
is meant (i.e. "and" instead of "and")?